shapely/tests/geometry/test_geometry_base.py

```diff
@@ -157,3 +158,110 @@ def test_constructive_properties(op):
     result = getattr(geom, op)
     expected = getattr(shapely, op)(geom)
     assert result == expected
+
+
+@pytest.mark.parametrize(
+    "op",
+    [
+        "crosses",
+        "contains",
+        # "contains_properly",
```

Comment:
to get `contains_properly` to work, perhaps add a try / except AttributeError with:
```python
fn = getattr(shapely.predicates, op)
result = fn(polygon, points)
...
expected = np.array([fn(polygon, p) for p in points], dtype=bool)
```
or perhaps just use that since each of these fn(a, b) signatures appear in the `shapely.predicates` submodule.

Reply:
To be clear, the reason `contains_properly` can't be tested here, is because that's not a method on the Geometry class (only a top-level function), xref https://github.com/shapely/shapely/issues/1281

So I should probably just do a PR to add `contains_properly` as a method.